exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/grpc/DefaultGrpcExporter.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.otlp.internal.grpc;
+
+import com.google.common.util.concurrent.FutureCallback;
+import com.google.common.util.concurrent.Futures;
+import com.google.common.util.concurrent.MoreExecutors;
+import io.grpc.Codec;
+import io.grpc.ManagedChannel;
+import io.grpc.Status;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.metrics.BoundLongCounter;
+import io.opentelemetry.api.metrics.GlobalMeterProvider;
+import io.opentelemetry.api.metrics.LongCounter;
+import io.opentelemetry.api.metrics.Meter;
+import io.opentelemetry.exporter.otlp.internal.Marshaler;
+import io.opentelemetry.sdk.common.CompletableResultCode;
+import io.opentelemetry.sdk.internal.ThrottlingLogger;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+/**
+ * A {@link GrpcExporter} which uses the standard grpc-java library.
+ *
+ * <p>This class is internal and is hence not for public use. Its APIs are unstable and can change
+ * at any time.
+ */
+public final class DefaultGrpcExporter<T extends Marshaler> implements GrpcExporter<T> {
+
+  private static final Logger internalLogger =
+      Logger.getLogger(DefaultGrpcExporter.class.getName());
+
+  private final ThrottlingLogger logger = new ThrottlingLogger(internalLogger);
+
+  private final String type;
+  private final ManagedChannel managedChannel;
+  private final MarshalerServiceStub<T, ?, ?> stub;
+  private final long timeoutNanos;
+
+  private final BoundLongCounter seen;
+  private final BoundLongCounter success;
+  private final BoundLongCounter failed;
+
+  /** Creates a new {@link DefaultGrpcExporter}. */
+  DefaultGrpcExporter(
+      String type,
+      ManagedChannel channel,
+      MarshalerServiceStub<T, ?, ?> stub,
+      long timeoutNanos,
+      boolean compressionEnabled) {
+    this.type = type;
+    Meter meter = GlobalMeterProvider.get().get("io.opentelemetry.exporters.otlp-grpc");
```

Comment:
Do we want to call out in the changelog that the name of these metrics are changing? The instrumentation library will change, and also the metric name will change from `io.opentelemetry.exporters.otlp` to `io.opentelemetry.exporters.otlp-grpc`.

Reply:
That's a good call, although this shouldn't change the name of any metrics (unless you have a custom exporter that appends the meter name to the metric name, I guess?). I would be slightly surprised if anyone was actually using these metrics yet, but in case they are, it is definitely reasonable to call it out.

Reply:
I renamed the meter names too, instead of having the type in the meter name it's an attribute now. The camelcase before also didn't seem conventional. I suspect calling out in release notes while making the change is good here - the instrumentation library change isn't that important so I could revert it, but given the metric name change anyways, I figured it's good to more easily differentiate metrics from -grpc, -grpc-okhttp, -http